test/index.html

```diff
@@ -15,7 +15,7 @@
         <script src="../reqs/contextmenu/leaflet.contextmenu-src.js"></script>
         <script src="../reqs/loading/Control.Loading.js"></script>
         <script src="../reqs/markercluster/leaflet.markercluster-src.js"></script>
-        <script src="../reqs/label/leaflet.label-src.js"></script>
+        <script src="../reqs/label/leaflet.label.js"></script>
```

Comment:
Leaflet.Label needs to be removed anyway to be replaced by L.Tooltip, see http://leafletjs.com/2016/07/16/leaflet-1.0-rc2.html